_posts/2025-02-27-use-desctop-visual-studio-code.adoc

```diff
@@ -0,0 +1,111 @@
+---
+title: "Working locally in the cloud: Develop in local Visual Studio Code with Eclipse Che as your Cloud Development Environment"
+layout: post
+author: Vitaliy Gulyy
+description: >-
+   Use your local Visual Studio Code as an editor for a Che workspace
+categories: []
+keywords: ['Visual Studio Code', 'Eclipse Che editor']
+slug: /@vgulyy/use-desktop-visual-studio-code
+---
+
+== Introduction
+
+Eclipse Che provides developers with a web version of the link:https://github.com/microsoft/vscode[Visual Studio Code OSS] editor, which includes most of the desktop version features. The transition to cloud development can feel quite unusual for developers, but Eclipse Che has a feature that will help you feel right at home immediately.
+With link:https://code.visualstudio.com/docs/editor/command-line[Visual Studio Code Command Line Interface (CLI)] you can create a tunnel that allows you to connect your familiar local Visual Studio Code to your workspace.
+
+== Connecting Visual Studio Code desktop to a new workspace
+
+You can connect your local Visual Studio Code to a new workpace with a few clicks.
+First, create a workspace on the Che Dashboard and choose Visual Studio Code (desktop) as your editor.
```

Comment:
```suggestion
First, create a CDE from the Che Dashboard and choose Visual Studio Code (desktop) as your editor.
```

Reply:
I suppose leaving `workspace` here will be more clear for the users. WDYT?

Reply:
Sounds good, it makes sense because the scerenshots (dashboard, code server) say `workspace`